src/shared/generator.c

```diff
@@ -298,8 +299,13 @@ int generator_write_fsck_deps(
                 }
         }
 
+        if (fstab_test_option(options, "bind\0")) {
```

Comment:
This fails to take rbind and negative opts into account? fstab_is_bind?

Reply:
what do you mean by negative opts in the context of "bind"?

Reply:
Oh, right, sorry for the noise. Some confusion in mind with other stuff...